repos/rust-lang/calendar.toml

```
@@ -4,6 +4,7 @@ description = "Calendars for Rust project teams"
 bots = ["rustbot"]
 
 [access.teams]
+cargo = "write"
```

```suggestion
cargo = "maintain"
```

consistent with all the other teams with hosted calendars

---

I did this because it was suggested [in this comment](https://github.com/rust-lang/team/pull/1180#discussion_r1446057760). I don't think `T-cargo` will use anything in `maintain`, we really only need `write`.

---

It's fine to keep things consistent and give cargo maintain privileges. We can then open a PR bumping everyone down to write.